app/assets/stylesheets/main.scss

```diff
@@ -10,6 +10,7 @@
   }
 
   .btn-sul-dlss:hover {
+    background-color: var(--cardinal-red);
     filter: brightness(85%);
     color: var(--bs-btn-color);
```

Comment:
This should be `--bs-btn-hover-color:`.   Both of these can go on `.btn-sul-dlss`